code/game/objects/barricades.dm

```diff
@@ -0,0 +1,379 @@
+/obj/structure/barrier
+	name = "bollards"
+	desc = "A barrier that allows humans to pass, but blocks vehicles. Not very durable."
```

Comment:
```suggestion
	desc = "A barrier that allows humans to pass, but blocks mechs. Not very durable."
```
Unless we get actual vehicles we should make clear it's about mechs.
The tug doesn't count it cannot even spawn.

Reply:
Touché.

Reply:
Aw, "vehicles" is better as a term because its generic, and mechs are just a subtype of a vehicle, it looks more professional imho because half these obstacles are based off of irl variants that just stop or deter cars